src/main/authentication/authentication.ts

```diff
@@ -123,8 +151,43 @@ function oidcSetup(): void {
     clientSecret: clientSecret,
     isB2C: true,
   },
-  passportStrategyFn,
+  pAndIVerifyFunction,
   ));
+
+
+  passport.use('cft-idam', new CustomStrategy(
```

Comment:
The CFT IDAM flow seems to be quite different to the AAD one. I wonder if it is worth creating a separate file called cftIdamAuthentication.ts or something similar and move the CFT code out to this new file

Reply:
Good idea, I've done this now